src/platform/mt8195/include/platform/lib/memory.h

```diff
@@ -165,7 +165,11 @@
 #define HEAP_BUF_ALIGNMENT DCACHE_LINE_SIZE
 
 /** \brief EDF task's default stack size in bytes. */
+#ifdef CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING
```

Comment:
Stack size should be taken from module manifest,
yes, it is not yet implemented :(

Reply:
Yeah, this is where the value goes for this platform still.  Working on zephyrization.